rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which section in the template make sense for their proposal. It's recommended to cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to retrospect later on the proposal.
+2. Add your RFC to the geode/rfcs source directory, update to the next unique number. 
+3. Post a PR for your RFC putting [RFC] in the title. Make sure to state the deadline clearly in your PR. 
+The duration of the deadline is up to the author(s) and should depend on the size and complexity of the proposal. A reasonable time frame might be somewhere between a week and a month.
+4. Answer questions and concerns on the PR. Consider adding questions that get asked more than once to the FAQ section of the RFC.
+5. After the deadline for feedback has been reached summarize the consensus and your decision on the PR  thread. 
+    1. If the decision is to drop the proposal, the status should be updated to Dropped. 
+    2. If we are moving forward the status goes to *Active*
+    3. When there is a newer RFC that replaces this one the status goes to *Superseded* and the *Superseded By* gets updated with the number of the new RFC.
+    
+###Immutability & Errata
+Once approved the existing body of the RFC should remain immutable. 
+
+Once the proposed work starts, we likely will learn things that require minor changes to the RFC to keep it current. These changes should be captured in an *Errata* section at the end of the document rather than modifying the existing body of the  document directly. This is to  highlight that and how the approach has changed, since initial discussion.
+
+### Humble Advice
+Some things can be helpful to keep in mind when writing technical documents:
+
+1. Keep the document brief but complete. People don’t have time to thoroughly read and think about extremely long documents and they’ll receive less feedback compared with a shorter document. If you find it challenging to meet this limit then maybe the proposal is too big and could be broken up.
+2. Include evidence of the problem if at all possible, even if it’s anecdotal. This can help others see the core causes of the issue rather than only being able to comment on the diagnosis or solution. For example, consider linking to evidence inline, brief inline quotes, and/or footnotes.
+3. IETF RFCs you may see contain strict rules conveyed within the semantic meaning of *SHOULD*, *MUST*, and *MAY*. You don’t need to stress about the particulars of language or semantics when writing Geode-RFCs. Place importance explaining your problem and proposal clearly, succinctly, and convincingly over the specifics of implementation.
+
+## Alternatives
```

Comment:
Given that this is a PR for one style of RFC, this `Alternatives` section should be removed. If necessary, it should just be a comment to the PR.

Reply:
I am a little torn on this. You are absolutely right that this is really part of the discussion, rather than part of the thing that's being proposed and thus could live as a comment. I do see some value in preserving context from the discussion in the committed artifacts itself. That's why I also propose the FAQ section. While we will likely have access to the PR for a long time, we might lose that context if we ever move away from GitHub.

This brings up a related question: The PR discussion provides valuable context. It might be nice to make it easily discoverable. What do we think about linking to the PR from the header? It means an additional step when creating the PR because of the circular dependency, but likely will safe some searching in the future.

Reply:
It might make more sense as a "prior art" section rather than alternatives.

Reply:
To elaborate, this section could list prior art and a rational for not using it. This way as others are reading the approved RFC later and wonder, "but why didn't we use X?", that perhaps X is referenced in prior art.